ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -1052,6 +1052,15 @@ periodics:
     ci.openshift.io/area: edge-infrastructure
     ci.openshift.io/role: image-mirroring
   name: periodic-image-mirroring-edge-infrastructure
+  reporter_config:
+    slack:
+      channel: '#forum-edge-infra-alerts'
+      job_states_to_report:
+      - failure
+      - error
+      report_template: '{{ if eq .Status.State "success" }} :green_jenkins_circle:
```

Comment:
The if-part seem never true if `success` is not included in `job_states_to_report`.

Reply:
Ah yeah this was adapted from a config we used to use that was more noisy. I updated it :)